homeassistant/components/airzone/manifest.json

```diff
@@ -6,5 +6,10 @@
   "requirements": ["aioairzone==0.4.5"],
   "codeowners": ["@Noltari"],
   "iot_class": "local_polling",
-  "loggers": ["aioairzone"]
+  "loggers": ["aioairzone"],
+  "dhcp": [
+    {
+      "macaddress": "E84F25*"
```

Comment:
Just realized this OUI is `Murata Manufacturing Ltd` which means it could match non-airzone devices.

Is there a hostname you can use as well?

Reply:
@bdraco unfortunately airzone doesn't set any hostname by default...

Reply:
> @bdraco unfortunately airzone doesn't set any hostname by default...

@bdraco However, I can ask Airzone if they can narrow down the mac address prefixes...